scalableminds/webknossos
# Add "description" attribute to annotations

### Mailable description of changes:
- Added support for  a free-text `description` field for annotations.
- Added  `description` support for NML files<things><parameters><experiment name="e2006_knossos" description="hallo johannes" /> ...
- [Tracing View] Added `description` input field to "Info Tab"
![image](https://user-images.githubusercontent.com/1105056/30810952-f8dcbf2e-a206-11e7-8429-4997f13c39c7.png)


### Steps to test:
1. Open a tracing
2. Edit the description. Check Network tab for request
3. Reload tracing and see description
4. Download skeletontracing as NML file and check for `description`
5. Upload a skeletontracing as NML with a test `description` and check that it is properly displayed in wK

### Issues:
- fixes #2041 

@daniel-wer Please review the frontend part.
@youri-k  Please review the backend part. 

------
